test/powershell/Modules/Microsoft.PowerShell.Utility/Start-Sleep.Tests.ps1

```diff
@@ -3,42 +3,50 @@
 Describe "Start-Sleep DRT Unit Tests" -Tags "CI" {
 
     # WaitHandle.WaitOne(milliseconds, exitContext) is not accurate.
-    # The wait time varies from 980ms to 1150ms from observation, so
-    # the tests here are changed to be greater than 950ms.
-    $minTime = 950
-    $maxTime = 1200
+    # The wait time varies from 980ms to 1150ms from observation
```

Comment:
The values is not correct. Please update the comment.

Reply:
I left the observations that were made with previous sleep time in the comment:
"The wait time varies from 980ms to 1150ms from observation when waiting for one second." 
and then how it is tested now:
 "To test fractional seconds the expected values start from 1450ms and end to 1700ms." 
How would you suggest the comment to be changed?

Reply:
You would replace " 980ms to 1150ms" with "1450ms to 1700ms"

Reply:
I would want to point out that to my understanding the values (980ms to 1150ms) have been derived by testing with sleep time of 1 second (as stated in the comment now). I use these observations to come up with the actual minimum and maximum values that are 500 milliseconds greater with a little safety margin. But I can change it if this is the right way to go.

Reply:
Now we don't use old values so the comment can confuse readers.

Reply:
This is changed now 👍